developer_docs/README.adoc

```diff
@@ -0,0 +1,18 @@
+= developer_docs
```

Comment:
It only depends on `rubygem-asciidoctor`. Perhaps also provide a hint to install just that?